shared/login/recover-password/device-selector/index.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as ProvisionTypes from '../../../constants/types/provision'
+import {globalMargins, styleSheetCreate, platformStyles, isMobile} from '../../../styles'
+import {SignupScreen} from '../../../signup/common'
+
+type ItemProps = {
+  index: number
+  item: {
+    name: string
+    type: string
+  }
+  onSelect: (name) => void
+  onResetAccount: () => void
+}
+
+const RecoverMenuItem = (props: ItemProps) => {
+  const {name, type} = props.item
+
+  if (type === 'account-reset') {
+    return (
+      <Kb.ListItem2
+        type="Small"
+        firstItem={props.index === 0}
+        key={name}
+        onClick={() => props.onResetAccount()}
+        icon={<Kb.Icon sizeType="Big" type="iconfont-skull" />}
+        body={
+          <Kb.Box2 direction="vertical" fullWidth={true}>
+            <Kb.Text type="BodySemibold">I don't have any of these</Kb.Text>
+            <Kb.Text type="BodySmall">Reset your account</Kb.Text>
+          </Kb.Box2>
+        }
+      />
+    )
+  }
+
+  let iconType
+  switch (type) {
+    case 'mobile':
+      iconType = 'icon-phone-32'
+      break
+    case 'desktop':
+      iconType = 'icon-computer-32'
+      break
+    case 'backup':
+      iconType = 'icon-paper-key-32'
+      break
+    default:
+      iconType = 'icon-paper-key-32'
+  }
+
+  return (
+    <Kb.ListItem2
+      type="Small"
+      firstItem={props.index === 0}
+      key={name}
+      onClick={() => props.onSelect(name)}
+      icon={<Kb.Icon type={iconType} />}
+      body={
+        <Kb.Box2 direction="vertical" fullWidth={true}>
+          <Kb.Text type="BodySemibold">{name}</Kb.Text>
+          {type === 'backup' && <Kb.Text type="BodySmall">Paper key</Kb.Text>}
+        </Kb.Box2>
+      }
+    />
+  )
+}
+
+export type Props = {
+  devices: Array<ProvisionTypes.Device>
+  onBack: () => void
+  onSelect: (name: string) => void
+  onResetAccount: () => void
+}
+
+const DeviceSelector = (props: Props) => {
+  const items = [
+    ...props.devices,
+    {
+      name: 'account-reset',
+      type: 'account-reset',
+    },
+  ]
+  return (
+    <SignupScreen banners={[]} buttons={[]} onBack={props.onBack} title="Recover password">
```

Comment:
`banners` and `buttons` are optional. In designs the body of this screen has a white background, currently it's blue

Reply:
use the `noBackground` prop here.